atlassian_jwt_auth/contrib/tests/django/test_django.py

```diff
@@ -0,0 +1,85 @@
+import django
+import os
+
+from django.test.testcases import SimpleTestCase
+from django.test.utils import override_settings
+
+from atlassian_jwt_auth import create_signer
+from atlassian_jwt_auth.contrib.tests.utils import get_static_retriever_class
+from atlassian_jwt_auth.tests import utils
+
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE',
```

Comment:
I feel that you may want this to go into a setup / tear down method (that isn't required but it might be preferable).